doc/isomorph.xml

```diff
@@ -41,7 +41,7 @@
 
 <#GAPDoc Label="BlissAutomorphismGroup">
 <ManSection>
-  <Attr Name="BlissAutomorphismGroup" Arg="digraph[, colours]"/>
+  <Attr Name="BlissAutomorphismGroup" Arg="digraph[, vertex_colours, edge_colours]"/>
```

Comment:
Does this have to be used with either both optional arguments or none, or is just vertex colours okay? In which case:
```suggestion
  <Attr Name="BlissAutomorphismGroup" Arg="digraph[, vertex_colours[, edge_colours]]"/>
```

Reply:
Just vertex colour is okay, fixed